universal-application-tool-0.0.1/app/views/admin/ProgramEditView.java

```diff
@@ -0,0 +1,31 @@
+package views.admin;
+
+import static j2html.TagCreator.body;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.join;
+
+import play.mvc.Http.Request;
+import play.twirl.api.Content;
+import services.program.ProgramDefinition;
+import views.BaseHtmlView;
+
+public class ProgramEditView extends BaseHtmlView {
+  public Content render(Request request, ProgramDefinition program) {
+    return htmlContent(
```

Comment:
same question here - does the `.with()` syntax help make this more readable?

Reply:
Let's do a follow-up PR to clean that up.